docs/core/accounting/SharesAccounting.md

```diff
@@ -47,47 +52,47 @@ $op_n = \sum_{i=1}^{k} s_{n,i}$ where the operator has $k$ number of stakers del
 
 $$
-    s_{n+1} = s_{n} + d_{n}
+ s_{n+1} = s_{n} + d_{n}
 $$
 
 $$
-    op_{n+1} = op_{n} + d_{n}
+ op_{n+1} = op_{n} + d_{n}
 $$
 
 ### Staker Withdrawals 
 
 Similarly for Staker withdrawals, given an amount $w_n$ to withdraw at time $n$, the Staker and Operator's shares are decremented at the point of the withdrawal being queued:
 
 $$
-    s_{n+1} = s_{n} - w_{n}
+ s_{n+1} = s_{n} - w_{n}
 $$
 
 $$  
-    op_{n+1} = op_{n} - w_{n}
+ op_{n+1} = op_{n} - w_{n}
 $$
 
-Later after the withdrawal delay has passed, the Staker can complete their withdrawal to withdraw the full amount $w_n$ of shares.
+Later after the withdrawal delay has passed, the Staker can complete their withdrawal to withdraw the full amount $w_n$ of shares. 
 
 
 ## Slashing Upgrade Changes
 
 As of release `v1.0.0` and the introduction of Unique Stake and Operator Sets, programmatic slashing will be enabled in the core EigenLayer protocol where Staker deposits can be subject to slashing. 
-The remaining portions of this document will assume understanding of Allocations/Deallocations, and Max Magnitudes, and OperatorSets.
-For more information on this, there is the [ELIP-002][elip-002] which has a high-level but detailed overview of the Slashing upgrade as well as the separate [Magnitude document][magnitude-doc] here.
+The remaining portions of this document will assume understanding of Allocations/Deallocations, Max Magnitudes, and OperatorSets.
+For more information on this, there is the [ELIP-002][elip-002] which has a high-level but detailed overview of the Slashing upgrade.
 
 We now introduce a few new types of Shares concepts:
 
 1. **deposit shares**: \
-    For a Staker, this is the amount of Strategy shares deposited
+ For a Staker, this is the amount of Strategy shares deposited
 2. **withdrawable shares**: \
-    For a Staker, this is the actual amount of Strategy shares they are eligible to withdraw. \
-    This does not live in storage but is read through the view function `DelegationManager.getWithdrawableShares`. Note that this amount is <= deposit shares as the Staker may have had their shares slashed.
+ For a Staker, this is the actual amount of Strategy shares they are eligible to withdraw. \
+ This does not live in storage but is read through the view function `DelegationManager.getWithdrawableShares`. Note that this amount is <= deposit shares as the Staker may have had their shares slashed.
 3. **operator/delegated shares**: \
-    This still remains the same definition as before, the amount of delegated shares of an operator from all their delegated stakers.
-    However, this is now equal to the summation of all their staker's withdrawable shares.
+ This still remains the same definition as before, the amount of delegated shares of an operator from all their delegated stakers.
+ However, this is now equal to the summation of all their staker's withdrawable shares.
```

Comment:
Since we already cover this in the terminology section, we should either refer to that section or reorder some sections so we don't repeat information